app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(
+            self,
```

Comment:
No need to have those two parameters start on a new line
It makes sense if there are more than two